tools/external_ic.F90

```diff
@@ -203,16 +206,12 @@ subroutine get_external_ic( Atm, fv_domain, cold_start )
         call prt_maxmin('graupel', Atm%q(:,:,:,graupel), is, ie, js, je, ng, Atm%npz, 1.)
         if ( o3mr > 0    ) &
         call prt_maxmin('O3MR',    Atm%q(:,:,:,o3mr),    is, ie, js, je, ng, Atm%npz, 1.)
+        if ( sgs_tke > 0    ) &
```

Comment:
Maybe later we should just cycle through all the tracer species instead of hard-coding each one.